airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/load/dataflow/state/StateStoreTest.kt

```diff
@@ -59,27 +59,6 @@ class StateStoreTest {
         verify { histogramStore.acceptExpectedCounts(stateKey, 100L) }
     }
 
-    @Test
-    fun `remove should return and remove the checkpoint message for given key`() {
```

Comment:
checking my understanding: there's no `remove` function anymore, so this test is pointless?